zend_parse_arg_impl.go

```diff
@@ -73,3 +66,52 @@ func zendParseArgAsString(value any) (string, error) {
 	}
 	return str, nil
 }
+
```

Comment:
When I see the `zend_parse_arg_impl` function from Zend/zend_API.c and the `_convert_to_cstring` function from Zend/zend_operators.c, it looks the purpose of those functions is different, so the two functions appear to be located in different files.

For example `_convert_to_cstring` function can also convert array and object to string, but inside the  `zend_parse_arg_impl` function, `_convert_to_cstring` is called only when `IS_STRING`, `IS_LONG`, `IS_DOUBLE`, and `IS_BOOL`. 

Meanwhile, the `_convert_to_cstring` function seems to be implemented for the purpose of turning everything into a string in a generalized way.

So how about to generate two files zend_api.go and zend_operators.go(or zend/api.go, zend/operators.go?) corresponding to zend_API.c and zend_operators.c.

Reply:
FYI It's `_convert_to_string` not `_convert_to_cstring`

Reply:
Sorry, I misspelled it. What I meant was `_convert_to_string ` and now I modified the function name to `_convert_to_string`

Reply:
I have separated the two functions into separate files: `zned_api.go` and `zend_operator.go`

Reply:
Thank you!